chaos_genius/connectors/base_db.py

```diff
@@ -4,9 +4,17 @@
 from sqlalchemy import exc as sqlalchemy_exc
 from sqlalchemy import text
 
+
 logger = logging.getLogger(__name__)
 
+
 class BaseDb:
+
+    SQL_IDENTIFIER = ""
+    SQL_DATE_TO_DATETIME_FORMAT = "T00:00:00{}"
+    SQL_STRPTIME_FORMAT = "%Y-%m-%dT%H:%M:%S%z"
+    SQL_STRFTIME_FORMAT = "%Y-%m-%dT%H:%M:%S"
```

Comment:
Docstrings for this would be useful for anyone implementing a connector.
```suggestion
    SQL_IDENTIFIER = ""
    """a doc string."""
    SQL_DATE_TO_DATETIME_FORMAT = "T00:00:00{}"
    """a doc string."""
    SQL_STRPTIME_FORMAT = "%Y-%m-%dT%H:%M:%S%z"
    """a doc string."""
    SQL_STRFTIME_FORMAT = "%Y-%m-%dT%H:%M:%S"
    """a doc string."""
```